scripts/cocoapods/helpers.rb

```diff
@@ -18,3 +18,9 @@ def ruby_platform
         return RUBY_PLATFORM
     end
 end
+
+class Finder
+    def self.find_codegen_file(path)
+        return `find #{path} -type f \\( -name "Native*.js" -or -name "*NativeComponent.js" \\)`.split("\n").sort()
+    end
```

Comment:
Hey @cipolleschi! I was wondering maybe adding TypeScript support for for local modules is a change small enough so this could be done here?

Reply:
Added. Thanks for the suggestion!